src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -67,7 +67,7 @@ func (gen *PythonStartupScriptGenerator) GenerateEntrypointScript() string {
 	scriptBuilder.WriteString("echo 'cd $APP_PATH' >> ~/.bashrc\n")
 
 	if gen.Configuration.EnableDynamicInstall && !common.PathExists(pythonInstallationRoot) {
-		scriptBuilder.WriteString(fmt.Sprintf("oryx setupEnv -appPath %s\n", gen.getAppPath()))
+		scriptBuilder.WriteString(fmt.Sprintf("oryx prep --src %s\n", appPath))
```

Comment:
why? i would not change this  for now

Reply:
Same as https://github.com/microsoft/Oryx/pull/1149/files/ce4f94e5a8fe1c05e3865f38118613f56f67d30d#r745944937

Reply:
I would   rather investigate why is it failing .. because prep is for different purpose, im guessing we somehow need to enable the setupEnv command in build image